WordPress/Classes/Utility/Editor/GutenbergSettings.swift

```diff
@@ -171,14 +174,22 @@ class GutenbergSettings {
 
     func canViewEditorOnboarding() -> Bool {
         guard
-            ReachabilityUtils.isInternetReachable(),
-            let account = AccountService(managedObjectContext: context).defaultWordPressComAccount()
+            ReachabilityUtils.isInternetReachable()
         else {
             return false
```

Comment:
Does it make sense to leverage the anonymous ID here rather than returning false if there is no internet connection? WDYT?

Reply:
I understand what you mean, but if I were to do it that way then the next time this block of code runs and the internet is available, the calculated rank will change for this user if they are a WPcom login and that would skew our analytics.

Reply:
Looking at this code, I wonder if it would be advisable to a) to completely remove the internet connection check and b) treat all users, online/offline/WP.com/non-WP.com, the same by just using `UUID().hashValue` instead of their WP.com user ID (in the case of WP.com users).

Reply:
Hrm, that makes perfect sense to me! I'll update it 👍

Reply:
@guarani updated in 4f623aa87923b37b99df26300b19064988ceb4d1